frontend/src/components/HacknightsParticipants.vue

```diff
@@ -6,11 +6,14 @@
           v-model="selectedParticipants"
           outlined
           :items="
-            getParticipants.filter(
-              participant =>
-                !getHacknight.participants.filter(y => y.id === participant.id)
-                  .length
-            )
+            getParticipants
+              .filter(
+                participant =>
+                  !getHacknight.participants.filter(
+                    y => y.id === participant.id
+                  ).length
+              )
+              .sort((a, b) => a.github.localeCompare(b.github))
```

Comment:
I know it wasn't perfect before but after adding another line this whole thing gets really unreadable and it clutters the template. Please move this logic to `computed`, perhaps as `participantSelectItems`.

It would also be cool to refactor filter function a bit, to make it more readable. Think this would do the work:

```js
participant => !getHacknight.participants.some(
    hacknightParticipant => hacknightParticipant.id === participant.id
)
```